mne/viz/epochs.py

```diff
@@ -864,7 +864,7 @@ def plot_epochs_psd(epochs, fmin=0, fmax=np.inf, tmin=None, tmax=None,
         Either "full" or "length" (default). If "full", the PSD will
         be normalized by the sampling rate as well as the length of
         the signal (as in nitime).
-    picks : array-like of int | None
+    picks : XXX
```

Comment:
tbd?

Reply:
Yes these are placeholders for once we converge